internal/readpassword/trezor.go

```diff
@@ -5,22 +5,86 @@ import (
 
 	"github.com/rfjakob/gocryptfs/internal/exitcodes"
 	"github.com/rfjakob/gocryptfs/internal/tlog"
+
+	"github.com/xaionaro-go/cryptoWallet"
+	"github.com/xaionaro-go/cryptoWallet/vendors"
 )
 
-// Trezor reads 16 deterministically derived bytes from a
+// It's required initial values to be constant to get the resulting key to
+// be deterministic (see the comment above and within function "Trezor()").
+// This values could be just a bunch of zeros, but it seems to be a little
+// degenerate case that may add risks, so random values were used.
+//
+// This values were generated using command:
+//   dd if=/dev/random of=/dev/stdout bs=48 count=1 2>/dev/null | hexdump -e '8/1 "0x%02x, " "\n"'
+var (
+	TREZOR_DUMMYKEY = []byte{
```

Comment:
The problem with a "random" value like this is that you can never know if it has been chosen for a particular reason. Like, maybe this exact value exposes a weakness? No one can tell.

Let's just go with 16 zeros. AES handles that just fine.

Reply:
I'm afraid of kinda randbow tables (but for encryption keys instead of hash function initial values) for zero-filled data. I mean somebody like NSA could have optimized ways of getting keys using known blocks of encrypted and decrypted data where the encrypted data is a zero-filled data. I understand that is sounds crazy, but I feel more secure to use a random data… On the other hand I agree with your argument (I had the same thought while generating that random numbers) and zero-filled data looks more trustfully. OK :)

Reply:
We can go for something like "gocryptfs.trezor" or "trezor.trezor.tr" or "tttttt..." if you prefer, I'm fine with that!

Reply:
Ok, that's sounds more reassuringly, thanks)